spec/moip2/entry_api_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe Moip2::EntryApi do
+  let(:entry_api) { described_class.new(sandbox_oauth_client) }
+
+
+  describe "show entries" do
+    let(:created_entries) do
+      VCR.use_cassette("create_entries") do
+        entry_api
+      end
+    end
+
+      it "returns all entries" do
```

Comment:
como vc vai separar em duas seções, vc nem precisa fazer os expectations em blocos assim. Dentro do bloco do `find_all`, como vc vai ter criado uma variável  com o `let` (por ex, `found_entries`), vc coloca:

`it { expect(found_entries.first.description).to eq "....." }`

Reply:
Um outro ponto que eu acho legal pra melhorar esses testes é fazer umas checagens mais abrangentes. Cheque pelo menos a existencia da maioria dos campos documentados da resposta (`amout`, `transferInstrument`, `status`, `events`).